Cython/Debugger/libcython.py

```diff
@@ -1172,7 +1172,10 @@ def invoke(self, args, from_tty):
             return
 
         local_cython_vars = cython_function.locals
-        max_name_length = len(max(local_cython_vars, key=len))
+        if local_cython_vars:
```

Comment:
In cases where you ran `cy locals` and there were none to report the call to `max` would fail with `ValueError: max() arg is an empty sequence`